yarn.lock

```diff
@@ -4258,6 +4258,31 @@
     "@formatjs/intl-localematcher" "0.5.4"
     tslib "^2.4.0"
 
+"@formatjs/ecma402-abstract@2.2.4":
```

Comment:
We have two copies of this in the lockfile now. I think we need to bump other related libraries in `package.json` so that we have everything in a lockstep and transitive dependencies are shared as much as possible.